IanGRap/Squish
# New Player Select Lobby!

Contributors: Austin

Goal:
- Port the new player select lobby into existing game

What Changed
- All new code can be found in the PlayerSelect folder

How to Test:
- Load scene New Lobby and make sure functionality works as expected
- Load New Lobby Match Scene and make sure that players are spawned in correctly with unique colors
- Test in standalone build
